Glyssen/Character/CharacterVerseData.cs

```diff
@@ -75,7 +77,21 @@ public static string GetStandardCharacterId(string bookId, StandardCharacter sta
 
 		public static bool IsCharacterOfType(string characterId, StandardCharacter standardCharacterType)
 		{
-			return characterId.StartsWith(GetCharacterPrefix(standardCharacterType));
+			return characterId.StartsWith(GetCharacterPrefix(standardCharacterType), StringComparison.Ordinal);
+		}
+
+		public static bool IsCharacterExtraBiblical(string characterId)
+		{
+			if (IsCharacterOfType(characterId, StandardCharacter.BookOrChapter))
+				return true;
+
+			if (IsCharacterOfType(characterId, StandardCharacter.ExtraBiblical))
+				return true;
+
+			if (IsCharacterOfType(characterId, StandardCharacter.Intro))
+				return true;
+
+			return false;
```

Comment:
Based on my speed testing, this method (which is not used) is much faster than our current implementation of IsCharacterStandard. I proposed we implement two methods:
		public static bool IsCharacterStandard(string characterId)
		{
			return IsCharacterOfType(characterId, StandardCharacter.Narrator) ||
				IsCharacterOfType(characterId, StandardCharacter.BookOrChapter) ||
				IsCharacterOfType(characterId, StandardCharacter.ExtraBiblical) ||
				IsCharacterOfType(characterId, StandardCharacter.Intro);
				// We could call IsCharacterExtraBiblical instead of the last three lines of this if,
				// but this is speed-critical code and the overhead of the extra method call is
				// expensive.
		}

		public static bool IsCharacterExtraBiblical(string characterId)
		{
			return IsCharacterOfType(characterId, StandardCharacter.BookOrChapter) ||
				IsCharacterOfType(characterId, StandardCharacter.ExtraBiblical) ||
				IsCharacterOfType(characterId, StandardCharacter.Intro);
		}
IsCharacterExtraBiblical should be called any place in the code where we currently call IsCharacterStandard with a false parameter. Any of the existing single-parameter calls (true) will continue to compile just fine using the single parameter version of that method.